cmd/puppeth/puppeth.go

```diff
@@ -47,8 +49,12 @@ func main() {
 		log.Root().SetHandler(log.LvlFilterHandler(log.Lvl(c.Int("loglevel")), log.StreamHandler(os.Stdout, log.TerminalFormat(true))))
 		rand.Seed(time.Now().UnixNano())
 
-		// Start the wizard and relinquish control
-		makeWizard(c.String("network")).run()
+		if !strings.Contains(c.String("network"), " ") && !strings.Contains(c.String("network"), "-") {
+			// Start the wizard and relinquish control
+			makeWizard(c.String("network")).run()
+		} else {
+			utils.Fatalf("No spaces or hyphens allowed in network name")
+		}
```

Comment:
It would be a bit cleaner if you placed the error handling in front:

```go
network := c.String("network")
if strings.Contains(network, " ") || strings.Contains(network, "-") {
    utils.Fatalf("No spaces or hyphens allowed in network name")
}
makeWizard(network).run()
```

Reply:
Done